ncm-metaconfig/src/main/metaconfig/haproxy/main.tt

```diff
@@ -0,0 +1,67 @@
+global
+[% FOREACH log IN global.logs -%]
+    log /dev/log [% FOREACH item IN log.value -%] [% item %] [%- END -%]
+
+    [%- END -%]
+
+[% FOREACH option IN global.config -%]
+    [% option.key %] [% option.value %]
+
+    [%- END -%]
+
+    [% IF global.stats.defined %]
+    stats [% FOREACH option IN global.stats.pairs %] [% option.key %] [% option.value %] [%- END -%]
```

Comment:
you do this also below, so maybe make a separate tt file `hash_to_string.tt` with
```
[% FOREACH pair IN data.pairs -%]
[%- option.key %] [% option.value -%]
[%- END -%]
```
and then do
```
[% IF global.stats.defined %]
stats [% INCLUDE 'metaconfig/haproxy/hash_to_string.tt' data=glbal.stats %]
[%- END %]
```